package.json

```diff
@@ -8,7 +8,7 @@
   "version": "0.52.1",
   "files": [
     "dist",
-    "lib/css/exports/*.less"
+    "lib"
```

Comment:
👍 This change looks ok to me. I ran `npm publish --dry-run` to see what was included and it all seems on the ball. Pasting here in case anyone is curious:

<details>
<summary>npm publish --dry-run results</summary>

```
> npm publish --dry-run

package: @stackoverflow/stacks@0.52.1
=== Tarball Contents ===
274.8kB dist/css/stacks.css
226.6kB dist/css/stacks.min.css
166.4kB dist/js/stacks.js
72.7kB  dist/js/stacks.min.js
1.5kB   package.json
364B    lib/tsconfig.json
4.8kB   lib/css/components/_stacks-avatars.less
3.9kB   lib/css/components/_stacks-badges.less
2.6kB   lib/css/components/_stacks-banners.less
707B    lib/css/components/_stacks-blank-states.less   
1.1kB   lib/css/base/_stacks-body.less
10.3kB  lib/css/atomic/_stacks-borders.less
908B    lib/css/components/_stacks-breadcrumbs.less
2.7kB   lib/css/components/_stacks-button-groups.less
17.4kB  lib/css/components/_stacks-buttons.less
1.3kB   lib/css/components/_stacks-cards.less
5.8kB   lib/css/components/_stacks-collapsible.less
4.3kB   lib/css/atomic/_stacks-colors.less
9.1kB   lib/css/base/_stacks-configuration-dynamic.less
3.1kB   lib/css/base/_stacks-configuration-static.less
10.8kB  lib/css/exports/_stacks-constants-colors.less
5.1kB   lib/css/exports/_stacks-constants-helpers.less
2.6kB   lib/css/exports/_stacks-constants-type.less    
679B    lib/css/exports/_stacks-exports.less
7.3kB   lib/css/atomic/_stacks-flex.less
9.9kB   lib/css/atomic/_stacks-grid.less
551B    lib/css/base/_stacks-icons.less
17.8kB  lib/css/components/_stacks-inputs.less
10.4kB  lib/css/base/_stacks-internals.less
3.6kB   lib/css/components/_stacks-link-previews.less
3.9kB   lib/css/components/_stacks-links.less
14.4kB  lib/css/atomic/_stacks-misc.less
4.0kB   lib/css/exports/_stacks-mixins.less
4.1kB   lib/css/components/_stacks-modals.less
1.9kB   lib/css/components/_stacks-navigation.less
4.8kB   lib/css/components/_stacks-notices.less
1.5kB   lib/css/components/_stacks-page-titles.less
1.5kB   lib/css/components/_stacks-pagination.less
8.0kB   lib/css/components/_stacks-popovers.less
5.5kB   lib/css/components/_stacks-progress-bars.less
1.6kB   lib/css/base/_stacks-reset-meyer.less
8.2kB   lib/css/base/_stacks-reset-normalize.less
468B    lib/css/base/_stacks-reset.less
6.5kB   lib/css/atomic/_stacks-spacing.less
3.4kB   lib/css/components/_stacks-spinner.less
9.0kB   lib/css/components/_stacks-tables.less
4.9kB   lib/css/components/_stacks-tags.less
2.6kB   lib/css/components/_stacks-toggle-switches.less
8.0kB   lib/css/atomic/_stacks-typography.less
2.4kB   lib/css/components/_stacks-user-cards.less
1.1kB   lib/css/components/_stacks-widget-dynamic.less
9.7kB   lib/css/components/_stacks-widget-static.less
10.2kB  lib/css/atomic/_stacks-width-height.less
1.3kB   lib/css/stacks-dynamic.less
2.1kB   lib/css/stacks-static.less
463B    lib/css/stacks.less
1.1kB   LICENSE.MD
2.6kB   README.md
31B     lib/ts/finalize.ts
7.0kB   lib/ts/controllers/s-expandable-control.ts
12.0kB  lib/ts/controllers/s-modal.ts
11.3kB  lib/ts/controllers/s-popover.ts
10.8kB  lib/ts/controllers/s-table.ts
8.4kB   lib/ts/controllers/s-tooltip.ts
816B    dist/js/stacks.d.ts
3.7kB   lib/ts/stacks.ts
222B    lib/ts/stimulus.d.ts
=== Tarball Details ===
name:          @stackoverflow/stacks
version:       0.52.1
package size:  183.4 kB
unpacked size: 1.1 MB
shasum:        097fd088d87e4af138df160dad3054a6daa429c2
integrity:     sha512-HW5qHdEcaqSUX[...]N0CeMEO4BcE+A==
total files:   67

+ @stackoverflow/stacks@0.52.1
```

</details>

No action needed.